_includes/head_custom.html

```diff
@@ -9,6 +9,10 @@
 
 <meta name="Keywords" content="GAP,group theory,computational group theory,algebra,computational algebra,computational discrete algebra,mathematical computation" />
 
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Ubuntu+Mono:ital,wght@0,400;0,700;1,400;1,700&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet">
```

Comment:
Ideally we should replace this and instead bundle the font inside the repository (to comply with GPDR -- innocent visitors to our website should not be tracked by Google just for a font.

I can update this PR to do that